CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [0.0.6] - 2021-05-27
+
+### Added
+- [PINK-247] Add a wrapper of the `/remind` endpoint for the "Envelopes" resource on the Signable API.
```

Comment:
* Test suite moved to GitHub Actions.
* Testing improved. VCR added, recordings made and documented

might as well mentioned what else was done in this version

Reply:
👍